src/utils/common/index.ts

```diff
@@ -0,0 +1,14 @@
+const capitalize = (str) => str.charAt(0).toUpperCase() + str.slice(1);
+
+const toCapitalizedList = (arr: string[]): string =>
+  arr.map(capitalize).join(', ');
+
+const capitalizeArrayOrString = (input: string[] | string): string =>
```

Comment:
`capitalizeArrayOrString` sound for me like it capitalize that can work with array or string, and it take some time for me for understand that it also join array into one string
`arr(capitalize).join(',')` pretty short by it self, do we really need special utill for this?

Reply:
True. 
What if 'joinAndCapitalizeItems()' ?